src/vs/workbench/contrib/files/browser/editors/fileEditorInput.ts

```diff
@@ -60,6 +60,10 @@ export class FileEditorInput extends AbstractTextResourceEditorInput implements
 			}
 		}
 
+		if (!(capabilities & EditorInputCapabilities.Readonly)) {
```

Comment:
Should this move into `AbstractResourceEditorInput` which is reused by other ediotrs?